IpmiFeaturePkg/Include/Policy/IpmiWatchdogPolicy.h

```diff
@@ -0,0 +1,23 @@
+/** @file
+  Definitions for the IPMI watchdog timers policy.
+
+  Copyright (c) Microsoft Corporation
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+
+**/
+
+#ifndef _IPMI_WATCHDOG_POLICY_H_
+#define _IPMI_WATCHDOG_POLICY_H_
+
+#include <uefi.h>
+
+typedef struct _IPMI_WATCHDOG_POLICY {
+  BOOLEAN    Frb2Enabled;
+  UINT8      Frb2TimeoutAction;
+  UINT16     Frb2TimeoutSeconds;
+  BOOLEAN    OsWatchdogEnabled;
+  UINT8      OsTimeoutAction;
+  UINT16     OsTimeoutSeconds;
+} IPMI_WATCHDOG_POLICY;
+

```

Comment:
I've been seeing policy headers go under `Include/Guid/*Policy.h` and include the policy guid in them. I don't have a big preference here, but just if we want to align on something or not.

Reply:
We probably should still include `gIpmiWatchdogPolicyGuid` here.

Reply:
As a matter of contention, we should probably decide on a standard. I don't particularly care if this live ins /Guid/, but figured /Policy/ might add some clarity. Thoughts?

Reply:
As a matter of contention, we should probably decide on a standard. I don't particularly care if this live ins /Guid/, but figured /Policy/ might add some clarity. Thoughts?